set.mm

```diff
@@ -5764,6 +5764,15 @@ use disjunction (although this is not required since definitions are
       ( wa anbi2d biancomd ) ADBFCDABCDEGH $.
   $}
 
+  ${
```

Comment:
can be used to shorten 19.28

Reply:
Although the proof can/will be shortened by the minimize command, I will add a "Proof shortened by" tag with your name (but will not create an OLD version).